src/config.rs

```diff
@@ -45,7 +45,7 @@ pub struct Config {
     /// Specify if port forwarding via UPnP should be done or not. This can be set to false if the network
```

Comment:
Perhaps "Specify if port forwarding via UPnP Internet Device Gateway Protocol (IGD) should be done or not..."